doc/nrf/device_guides/working_with_nrf/nrf53/qspi_xip_guide.rst

```diff
@@ -0,0 +1,324 @@
+.. _qspi_xip:
+
+Configuring external XIP
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes the external XIP support for the nRF5340 SoC.
+The nRF5340 SoC is equipped with QSPI memory interface which is capable of exposing QSPI flash as memory for CPU used to fetch and execute the program code.
+The QSPI is available for the application core.
+Therefore, it is possible to relocate a part of the application's code to an external memory.
+NCS supports dividing an application into an internal and external part, along with the MCUboot support.
+
+Project setup
+*************
+
+To configure a project for QSPI XIP, you need to perform the following steps specific to the ext-xip feature:
+
+ * Provide QSPI flash memory DTS.
+ * Proviode a dedicated static partition manager configuration file with proper internal and external partitioning.
+ * Provide a linker file that describes the memory block of the QSPI XIP code location.
+ * Describe code relocation in the project's CMake file.
+
+Additionally, regular project setup elements are mandatory, for example, an application for the application core and a networking stack for the networking core.
+You will need to further configure those elements.
+
+Kconfig
```

Comment:
Perhaps this can be renamed to "Enable configuration options" or "Adding configuration options" ?

Reply:
Agreed!